regulations/static/regulations/css/scss/layout.scss

```diff
@@ -15,7 +15,7 @@ Main Content
 
 .main-content {
     background: $white;
-    margin-top: $mainhead_height + $subhead_height;
+    margin-top: $mainhead_height + $subhead_height + 4px;
```

Comment:
Quick check: was this `4px` intentional? If so, why is it needed (please add as a comment)